src/components/shared/marginals/footer.jsx

```diff
@@ -0,0 +1,83 @@
+import footerData from '../../../data/footerData.js';
+import { Heading, Paragraph } from '../typography/index.jsx';
+
+export default function Footer() {
+	return (
+		<div className='w-full h-[475px] bg-[#1E1E1E]'>
+			<div className='w-[95%] mx-auto md:pt-[16px] md:pb-[52px]' style={{ borderBottom: '1px solid white' }}>
+				<div className='w-full flex justify-end'>
```

Comment:
A gap can be added.
![image](https://github.com/dscnitrourkela/project-udon/assets/100484401/ec41fe23-0ab0-4c94-a594-65989dea4df5)